README.md

```diff
@@ -4,15 +4,31 @@
-| blockchain-link   | [blockchain-link](./packages/blockchain-link)                                                                                                                                                     |
-| integration-tests | [integration-tests](./packages/integration-tests)                                                                                                                                                 |
-
-## Development
+| package                                                   | description                                 |
+| --------------------------------------------------------- | ------------------------------------------- |
+| [@trezor/blockchain-link](./packages/blockchain-link)     | lib for connecting to blockchains           |
+| [@trezor/components](./packages/components)               | frontend react components                   |
+| [@trezor/connect-common](./packages/connect-common)       | static files and commons for trezor-connect |
+| [@trezor/connect-explorer](./packages/connect-explorer)   | interactive demo for trezor-connect         |
+| [@trezor/connect-iframe](./packages/connect-iframe)       | connect-iframe build from monorepo          |
+| [@trezor/integration-tests](./packages/integration-tests) | cross-packages e2e tests                    |
+| [@trezor/news-api](./packages/news-api)                   | medium proxy providing allow-origin headers |
+| [@trezor/rollout](./packages/rollout)                     | firmware updated utilities                  |
+| [@trezor/suite-build](./packages/suite-build)             | build utilities                             |
+| [@trezor/suite-data](./packages/suite-data)               | suite static data                           |
+| [@trezor/suite-desktop-api](./packages/suite-desktop-api) | API for suite - suite-desktop communication |
+| [@trezor/suite-desktop](./packages/suite-desktop)         | suite build target for Mac, Win, Linux      |
+| [@trezor/suite-native](./packages/suite-native)           | suite build target for react-native         |
+| [@trezor/suite-storage](./packages/suite-storage)         | abstract database definition for suite      |
+| [@trezor/suite-web-landing](./packages/suite-web-landing) | https://suite.trezor.io/                    |
+| [@trezor/suite-web](./packages/suite-web)                 | suite build target for web                  |
+| [@trezor/suite](./packages/suite)                         | trezor suite common code                    |
+| [@trezor/transport-native](./packages/transport-native)   | communication lib for react-native          |
+| [@trezor/transport](./packages/transport)                 | communication lib for javascript            |
+| [@trezor/utils](./packages/utils)                         | shared utility functions                    |
+| [@trezor/utxo-lib](./packages/utxo-lib)                   | btc-like coins lib                          |
+
+## @trezor/suite development
```

Comment:
renamed this section. maybe it could be moved into respective package readme files later.